pyproject.toml

```diff
@@ -0,0 +1,145 @@
+[build-system]
+requires = ['hatchling']
+build-backend = 'hatchling.build'
+
+[tool.hatch.version]
+path = 'pydantic/version.py'
+
+[project]
+name = 'pydantic'
+description = 'Data validation using Python type hints'
+authors = [
+    {name = 'Samuel Colvin', email = 's@muelcolvin.com'},
+    {name = 'Eric Jolibois', email = 'em.jolibois@gmail.com'},
+    {name = 'Hasan Ramezani', email = 'hasan.r67@gmail.com'},
+]
+license = {file = 'LICENSE'}
+readme = 'README.md'
```

Comment:
This stop adding the changelog to the readme on release, but I think that's fine since we're linking to the changelog with `[project.urls]`